apps/cli/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+import { join } from "node:path";
+import envPaths from "env-paths";
+
+const APP_CONFIG = envPaths("webstudio").config;
+const GLOBAL_CONFIG_FILE_NAME = "webstudio-config.json";
```

Comment:
why not just webstudio.json ?

Reply:
This is for global config. On the local level for the project it can be just `webstudio.json`. Or both can be the same,